packages/react/src/input/Input.tsx

```diff
@@ -9,16 +9,25 @@ import * as styles from "./Input.css";
 type InputProps = ExtendProps<
   ComponentPropsWithRef<"input">,
   ComponentPropsWithRef<typeof Box>,
-  { isDisabled?: boolean; isInvalid?: boolean } & styles.InputVariants
+  {
+    isDisabled?: boolean;
+    isInvalid?: boolean;
+    leftSection?: ReactNode;
+    rightSection?: ReactNode;
+  } & styles.WrapperVariants
 >;
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
   (
     {
       className,
+      defaultValue,
+      id,
       isDisabled,
       isInvalid,
```

Comment:
let's rename this from `isInvalid` to `error`. there is not `error` html attribute - but this is just to remain consistent with other attributes like `disabled`, `readonly`, etc

Reply:
Okay vai